src/main/java/com/rybka/Application.java

```diff
@@ -1,22 +1,35 @@
 package com.rybka;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import com.rybka.dao.HibernateDAO;
-import com.rybka.service.BaseCurrencyExchangeConnector;
-import com.rybka.service.ExchangeService;
 import com.rybka.service.ExchangeRateConnector;
+import com.rybka.service.ExchangeService;
+import com.rybka.service.PrimeExchangeRateConnector;
 import com.rybka.view.ExchangeView;
 import coresearch.cvurl.io.request.CVurl;
 
+import java.net.http.HttpClient;
+import java.util.HashMap;
 import java.util.Scanner;
 
 public class Application {
     public static void main(String[] args) {
 
-        BaseCurrencyExchangeConnector baseCurrencyExchangeConnector = new ExchangeRateConnector(new CVurl());
+        var reader = new PropertyReader(
```

Comment:
PropertyReader shouldn't know any thing about Map with services. 
Remove that dependency, build map in separate method. 

var reader = new PropertyReader("property.file")
var currencyConnectorsMap = buildMap()

var connector = currencyConnectorsMap.get(reader.getProperty('some.property')); 

In result, 
1. PropertyReader would't have dependency. 
2. PR will be more reusable. 

Read about Single Responsibility Principle(SOLID)

Reply:
fixed